src/CognitoCustomMessageProcessor/Contracts/IParticipantRegistrationDynamoDbRepository.cs

```diff
@@ -0,0 +1,8 @@
+using System.Threading.Tasks;
+
+namespace CognitoCustomMessageProcessor.Contracts;
+
+public interface IParticipantRegistrationDynamoDbRepository
+{
+    Task<string> GetParticipantLocaleAsync(string participantId);
```

Comment:
Do we have similar needs in web app? Any value in sharing code via common?

Reply:
There is no real use case I can think of where we would only be interested in the locale alone in the web app, we get it as part of the ParticipantDetails